config/theme.go

```diff
@@ -84,5 +84,45 @@ func GetColor(key string) tcell.Color {
 func GetColorString(key string) string {
 	themeMu.RLock()
 	defer themeMu.RUnlock()
-	return fmt.Sprintf("#%06x", theme[key].TrueColor().Hex())
+	color := theme[key].TrueColor()
+	if color == tcell.ColorDefault {
+		return "-"
+	}
+	return fmt.Sprintf("#%06x", color.Hex())
+}
+
+// GetReadableColor Returns ColorBlack if the color is brighter than gray
+// otherwise returns ColorWhite
+func GetReadableColor(bg tcell.Color) tcell.Color {
+	if bg == tcell.ColorDefault {
+		return tcell.ColorWhite
```

Comment:
Doesn't this assume that the default background is dark, so white text would be a good choice? This comes to the main problem with #181, which is knowing the user's terminal (and text) color. Once that is solved (and it can be, I just haven't implemented it yet), everything else can be done.

Reply:
yes it does, but because we reject any text fields with a value of `"default"`, and the `bg` argument is always set to a string ending in `"text"`, the if condition is never true. I just added that because `tcell.ColorDefault.RGB()` returns `-1, -1, -1`, so if this function was used in the future, I didn't want unexpected calculations to run if tcell.ColorDefault was passed in.

Now that I think about it, a better solution might be to return `tcell.ColorDefault`, but I'll let you decide

Reply:
Oh, I see. Could you rename `bg` to be more appropriate, seeing as it's not backgrounds, but like text colors, or rather the backgrounds being set to the color of the text? To be honest maybe the function should be renamed to `GetBlackWhite`, and `bg` to `c`, with comments that explain what it does, and the case for `tcell.ColorDefault`.

> Now that I think about it, a better solution might be to return `tcell.ColorDefault`, but I'll let you decide

I think that makes more sense. Please also add a comment  in the function explaining why this shouldn't happen anyway.